test/test-authorizedRequestFactory.js

```diff
@@ -87,9 +105,117 @@ describe('utils.authorizedRequestFactory', function() {
           assert.ok(typeof response === 'object');
           assert.deepEqual(body, 'test');
           mock.done();
+          process.env.GOOGLE_APPLICATION_CREDENTIALS = oldEnv;
           done();
         }
       );
     });
+    it('should take config input containing credentials without throwing', function (done) {
+      var utils = require('../lib/utils.js');
+      var config = {
+        credentials: require(path.join('..', validCredentialsPath)) // require is relative to this file
+      };
+      var req = utils.authorizedRequestFactory(['https://www.googleapis.com/auth/cloud-platform'],
+        config);
+      var mock = nock('http://www.test.com')
+        .get('/test')
+        .once()
```

Comment:
I believe `once` is the default behavior of `get`. Do you observe a behavior difference without it?